.npmignore

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
I though this change was for a `.npmrc` file or is this the same?

Reply:
cool :+1:

Reply:
lol, sorry :P